datafusion/sql/src/planner.rs

```diff
@@ -438,7 +438,7 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
             }
             SQLDataType::Bytea => Ok(DataType::Binary),
             SQLDataType::Interval => Ok(DataType::Interval(IntervalUnit::MonthDayNano)),
-            SQLDataType::Struct(fields) => {
+            SQLDataType::Struct(fields, _) => {
```

Comment:
Likewise here I think it would be good to name the field like

```suggestion
            SQLDataType::Struct(fields, _bracket_type) => {
```

So it is easier to understand by reading just the code what is being ignored